mod.json

```diff
@@ -2,15 +2,15 @@
   "_QPVersion": "0.1.1",
   "id": "BeatLeader",
   "name": "BeatLeader",
-  "version": "0.1.1",
+  "version": "1.0.1",
```

Comment:
Sorry, missed that. The last version was 0.1.0, this one is 0.1.1. If you don't like - use the 0.2.0, but not 1.0.1, it will mean it's released